lib/sync/endpoint.php

```diff
@@ -0,0 +1,210 @@
+<?php
+/**
+ * Adds an http based signaling server for real time collaboration.
```

Comment:
Nitpick:
```suggestion
 * Adds a HTTP-based signaling server for real time collaboration.
```

Reply:
Could be further improved if we get nitpicky:

```suggestion
 * Adds an HTTP-based signaling server for real-time collaboration.
```